pesto-cli/pesto/cli/test.py

```diff
@@ -5,10 +5,10 @@
 from pesto.common.testing.test_runner import TestRunner
 
 
-def test(build_config_path, profiles, nvidia=False):
-    build_config = BuildConfig.from_path(path=build_config_path, profiles=profiles)
+def test(build_config_path, profiles, nvidia=False, network="host"):
+    build_config = BuildConfig.from_path(path=build_config_path, profiles=profiles, network=network)
```

Comment:
build_config is just created to get docker image name, I think you don't have to pass `network=network` to the build_config. This is actually a network to run tests, not for building.

Reply:
Yes but I thought it will be interesting to add this option for CI purpose to run `pesto test` with a specifi network.

Reply:
Agreed, my point was: you can call `BuildConfig.from_path()` without giving the network kwarg because here you will run pesto test, not compile the docker. But it is also safe to leave it here.

Reply:
Oh okay got it. Sorry, I misunderstood your last comment. You are right there's not need to pass network as a parameter to Build_config.